### ISSUE TITLE
Why weights are 0 when output_occ < 0?

### ISSUE THREAD

**Issue by wtiandong** [angeladai/spsg#14]
Hi,
I notice [here](https://github.com/angeladai/spsg/blob/8d0dff65480116799cf35499a7f672c2360bec60/torch/train.py#L343)
you zero the weight where predicted output_occ is less than 0 (sigmoid(output_occ) < 0.5). If all output_occs are less than 0, all weights are 0. That leads to loss_occ = 0. Could you please give some hints about this question?

```python
if output_occ is not None:
    empty = torch.nn.Sigmoid()(output_occ.detach()) < 0.5
    weight[empty] = 0
    loss_occ = loss_util.compute_geo_occ_loss(target_for_sdf, output_occ, known, weight, args.truncation)
```
